spec/features/equipment_model_views_spec.rb

```diff
@@ -3,9 +3,68 @@
 describe 'Equipment model views' do
   subject { page }
 
+  shared_examples 'can view detailed table' do
+    it { should have_selector 'th', text: 'Available' }
+  end
+
+  shared_examples 'can view full and detailed andtable' do
```

Comment:
This is great, really nice work! Just a small typo with the title of this shared example, I'm guessing you meant "...detailed table" instead.